docset/windows/smbshare/Grant-SmbShareAccess.md

```diff
@@ -45,7 +45,7 @@ The **Grant-SmbShareAccess** cmdlet adds an allow access control entry (ACE) for
 ## EXAMPLES
 
 ### Example 1: Adds an allow ACE for a trustee
```

Comment:
```suggestion
### Example 1: Adds an allows ACE for a trustee
```

Reply:
It is allow access control entry (ACE). I believe allow is correct here.